server/queries/index.js

```diff
@@ -31,6 +31,7 @@ export default class Queries {
       .select('*')
       .from('pull_request_review_requests')
       .orderBy('created_at', 'asc')
+      .where('archived_at', null)
```

Comment:
we add a where statement here too for prrrs that have a null value at 'created_at'.